build/media_source/plg_system_shortcut/js/shortcut.es6.js

```diff
@@ -103,11 +103,11 @@
     let dl = '<dl>';
     dlItems.forEach((titles, shortcut) => {
       dl += '<dt><kbd>J</kbd>';
-      shortcut.split('+').forEach((key) => {
-        dl += ` <kbd>${key.trim()}</kbd>`;
+      shortcut.split('+').forEach(key => {
+        dl += ` ${Joomla.Text._('PLG_SYSTEM_SHORTCUT_THEN')} <kbd>${key.trim()}</kbd>`;
       });
       dl += '</dt>';
-      titles.forEach((title) => {
+      titles.forEach(title => {
```

Comment:
```suggestion
      titles.forEach((title) => {
```
Fix JS code style error reported by drone.